ps/jenkins/percona-server-for-mysql-8.0.groovy

```diff
@@ -656,9 +770,17 @@ parameters {
 
                 """
                 }
+
+                echo "Start Minitests for PS"
+                
+                package_tests_ps80()
+                
                 echo "Trigger Package Testing Job for PS"
+
                 build job: 'package-testing-ps80', propagate: false, wait: false, parameters: [string(name: 'product_to_test', value: 'ps80'),string(name: 'install_repo', value: "testing"),string(name: 'node_to_test', value: "all"),string(name: 'action_to_test', value: "all"),string(name: 'check_warnings', value: "yes"),string(name: 'install_mysql_shell', value: "no")]
```

Comment:
Are we going to keep triggering the job and running the same playbook in the job?